docs/components/button-group.md

```diff
@@ -1,6 +1,7 @@
 <script setup>
-import FwbButtonGroupExample from './buttonGroup/examples/FwbButtonGroupExample.vue'
-import FwbButtonGroupExampleIcon from './buttonGroup/examples/FwbButtonGroupExampleIcon.vue'
+import FwbButtonGroupExample from './buttonGroup/examples/FwbButtonGroupExample.vue';
+import FwbButtonGroupExampleIcon from './buttonGroup/examples/FwbButtonGroupExampleIcon.vue';
+import FwbButtonGroupExampleDropdown from './buttonGroup/examples/FwbButtonGroupExampleDropdown.vue';
```

Comment:
please remove semicolons ;)

Reply:
Thanks for the hint, I just fixed it. :)